.eslintrc

```diff
@@ -13,6 +13,7 @@
     ],
     "rules": {
       "no-empty-function": "off",
-      "@typescript-eslint/no-empty-function": "off"
+      "@typescript-eslint/no-empty-function": "off",
+      "@typescript-eslint/no-explicit-any": "off"
```

Comment:
@mwarzybok-sumoheavy if possible, let's not disable `@typescript-eslint/no-explicit-any`. If we're only using this for errors, we could use `catch (e: unknown)` instead. Then, we'll need to check `typeof` before operating on it.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#unknown-on-catch-clause-bindings

I did see that we've added `"useUnknownInCatchVariables": true,` and it has a similar effect but different use case:

> This pattern ensures that error handling code becomes more comprehensive because you cannot guarantee that the object being thrown is a Error subclass ahead of time. With the flag useUnknownInCatchVariables enabled, then you do not need the additional syntax (: unknown) nor a linter rule to try enforce this behavior.

https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables

Reply:
I didn't want to make code more complex (ifology), in previous version it was also on "false" but we didn't inform about it in tsconfig, so nothing has changed. Now it's only more clear what rules do we use.  I can use use if statement but number lines of code will increase without any important advantages. We know that in "e" we will have exception so it doesn't matter which type of exception we have because we are using only e.message from it.